### DISCUSSION TITLE
Display bug

### DISCUSSION THREAD

**Discussion by Monque27** [Androp0v/BioViewer#11] [General]
Just downloaded the app from Apple's App Store, running it on a MBP 2017. 
I'm getting a display bug where the program renders the atoms as squares (or something... see attached pic

<img width="750" alt="CleanShot 2022-04-06 at 17 34 41@2x" src="https://user-images.githubusercontent.com/61249900/162076547-a2e76bf5-5649-441f-b909-18067d6b0bfc.png">
)

**Comment by Androp0v** [comment]
Well, that's not ideal. 😅

I have a 2020 Intel MBP available now, so I can try to reproduce the issue and fix it this weekend. If I can reproduce the bug, I'd expect the fix to be live on the AppStore sometime next week. I'll update this thread with whatever I find.

**Reply by Monque27** [reply]
Thanks! 👍

**Reply by Androp0v** [reply]
A fixed version is on the AppStore now!

**Comment by Androp0v** [comment]
Looks like it broke on f97693c18d77f5192dce0f98f862c9ce1b0af028. It's a bit weird, 'cause it only introduces minimal shader optimizations, and it doesn't make sense that any of those minimal changes can be breaking, but... \*shrug*

I'll try to understand what's happening, but I'll send a version to Apple Store review tonight in any case (already have a fixed version working).
